tenant-file/portal-app/src/component/tenant-list.tsx

```diff
@@ -131,7 +144,7 @@ const TenantList: React.FC = () => {
           prepareRow(row);
           return (
             <>
-              <tr {...row.getRowProps()}>
+              <tr {...row.getRowProps()} onClick={onHandleRowClick} data-user-id={mockUserIdNumber}>
```

Comment:
Can you use styled-components to add a `cursor: pointer;` to the row?

Reply:
I had added hover effect (line 132) to table. Do you want to leave the hover shading as well as pointer?
I Just added cursor at 147: 
<tr style={{cursor:'pointer'}} {...row.getRowProps()} onClick={onHandleRowClick} data-user-id={mockUserIdNumber}>